File: Python/66. Plus One.py
class Solution(object):
    def plusOne(self, digits):
        """
        :type digits: List[int]
        :rtype: List[int]
        """
        a = "".join(map(str, digits))
        a = int(a) + 1
        b = list(str(a))
        return b

File: C++/66. Plus One.cpp
class Solution {
public:
    vector<int> plusOne(vector<int>& digits) {
        int overflow = 1, temp;

        for (int i = digits.size() - 1; i >= 0; i--) {
            temp = digits[i] + overflow;
            digits[i] = temp % 10;
            overflow = temp / 10;
        }

        if (overflow != 0) digits.insert(digits.begin(), overflow);

        return digits;
    }
};
